PcsConfig/SeedDataTest.cs

```diff
@@ -15,10 +15,18 @@ public class SeedDataTest
         private const string CONFIG_ADDRESS = "http://127.0.0.1:9005/v1";
         private const string TELEMETRY_ADDRESS = "http://127.0.0.1:9004/v1";
         private const string SIMULATION_ADDRESS = "http://127.0.0.1:9003/v1";
+        private const string STORAGE_ADAPTER_ADDRESS_FORMAT = "http://{0}:9022/v1";
+        private readonly string STORAGE_ADAPTER_ADDRESS;
+        private const int TIMEOUT = 5000;
 
         public SeedDataTest()
         {
+            // Wait for config to run seed data
```

Comment:
nit: the comment is in the wrong place, and not needed really because the function name is self-explanatory

Reply:
done